src/grandorgue/GOOrganController.h

```diff
@@ -73,6 +72,7 @@ class GOOrganController : public GOEventDistributor,
   GOAudioRecorder *m_AudioRecorder;
   GOMidiPlayer *m_MidiPlayer;
   GOMidiRecorder *m_MidiRecorder;
+  GOTimer *m_TimerManager;
```

Comment:
```suggestion
  GOTimer *m_TimerManager;
```

Either rename `m_TimerManager` to `m_timer` or rename `GOTimer` to `GOTimerManager`